any_member.hpp

```diff
@@ -545,6 +545,9 @@ class MemberSymbolTable
     bool equals(MemberSymbolTable<ClassType> const&) const;
     std::vector<std::string> const& member_names() const;
 
+    size_t get_member_index(std::string const&) const;
```

Comment:
We probably should use `int` here now as lmi moved away from using `size_t` since the original PR was done.